cl/opinion_page/tests.py

```diff
@@ -1,6 +1,13 @@
 from django.conf import settings
 from django.urls import reverse
 from django.test import TestCase, override_settings
+from django.test.client import Client
+from django.contrib.auth.models import User, Group
+from django.core.files.uploadedfile import SimpleUploadedFile
+
+from .forms import TennWorkersForm
```

Comment:
No relative imports in CL, please. They cause pain later.

Reply:
√